packages/smart-wallet/test/test-addAsset.js

```diff
@@ -495,13 +507,15 @@ test.serial('non-vbank asset: give before deposit', async t => {
       proposal: { give, want },
     });
     t.log('goofy client: propose to give', choices.join(', '));
-    await E(walletBridge).proposeOffer(ctx.fromBoard.toCapData(offer1));
+    await t.throwsAsync(
+      () => E(walletBridge).proposeOffer(ctx.fromBoard.toCapData(offer1)),
+      { message: /Withdrawal of .* failed because the purse only contained/ },
```

Comment:
why did this change? I suspect it's not necessary

Reply:
Please revert the change error checking change. I confirmed it's not necessary using,
```
git checkout master test/test-addAsset.js
yarn test test/test-addAsset.js
```